ts/nni_manager/common/trainingService.ts

```diff
@@ -18,12 +18,31 @@ interface HyperParameters {
     readonly index: number;
 }
 
+type PlacementConstraintType = 'None' | 'GPUNumber' | 'Device'
+interface PlacementConstraint{
+    readonly type: PlacementConstraintType;
+    readonly gpus: Array<number> | Array<[string,number]>;
+    /**
+     * GPUNumber constraint is in form of Array<number>, e.g., [3] means it must be placed on a node of 3 GPUs
+     * 
+     * Device constraint is in form of Array<[string,number]>, e.g., [('Node-0',1),('Node-0',0)] means it must be placed on 
```

Comment:
[('Node-0',1),('Node-0',0)] -> [('Node-0',1),('Node-1',0)]?

Reply:
It's a typo. Fixed.